src/hooks/useDebounce.js

```diff
@@ -1,6 +1,12 @@
 import {useEffect, useRef} from 'react';
 import _ from 'underscore';
 
+/**
+ * @param {*} func Function to debounce `waitMS` ms.
+ * @param {*} wait The number of milliseconds to wait before `func` can be invoked again.
+ * @param {*} immediate True if `func` should be invoked on the leading edge of `waitMS` instead of the trailing edge.
```

Comment:
Actually we [have to use Lodash](https://github.com/Expensify/App/pull/27490/files/d9ab01755e8a016cecc814fe18aa4dbdc9429508..3a577e623ab57f10a8ee719656b5ab1b9609b8d1#r1331217292)